views/make/index.js

```diff
@@ -100,6 +100,13 @@ module.exports = view.extend({
             self.$dispatch('openModalPrompt');
         });
 
+        self.$on('sideMenuShareClick', function (event) {
+            document.location =
```

Comment:
you don't want `document.location` here, this will cause a page refresh -- require `page` and use `page('/make...');`